plugins/dependencies/src/main/kotlin/org/gradle/kotlin/dsl/IsNotADependency.kt

```diff
@@ -5,6 +5,11 @@ package org.gradle.kotlin.dsl
 import org.gradle.api.artifacts.Dependency
 import org.gradle.api.artifacts.dsl.DependencyHandler
 
+/**
+ * Marker interface that prevents the user to add a wrong dependency like
+ *     implementation(AndroidX.core)
+ * Since AndroidX.core IsNotADependency, the IDE will show a warning
```

Comment:
`AndroidX.core` is a dependency, we need to use another example.

Reply:
I updated the example